packages/components/pie-modal/src/index.ts

```diff
@@ -165,34 +179,24 @@ export class PieModal extends RtlMixin(LitElement) {
             event.clientX < left ||
             event.clientX > right;
 
-        if (isClickOutsideDialog) {
+        if (isClickOutsideDialog && this.isDismissible) {
             this.isOpen = false;
         }
     };
 
     /**
-     * Dispatch `ON_MODAL_CLOSE_EVENT` event.
-     * To be used whenever we close the modal.
+     * Note: We should aim to have a shareable event helper system to allow
```

Comment:
Is there a ticket for this? Even if we don't fully define or refine it yet it would be good to have it created.

Reply:
Maybe as part of the session we have today/tomorrow? I've noted it down.